modules/library/render/src/test/java/org/geotools/renderer/lite/LabelFontShrinkSizeMinTest.java

```diff
@@ -0,0 +1,129 @@
+import org.geotools.styling.Style;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Polygon;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+
+/**
+ * Test class for verifying behaviour of displacementMode vendor option for both single layer and
+ * multi-layer map
+ *
+ * @author nprigour
+ */
+public class LabelFontShrinkSizeMinTest extends TestCase {
+
+    private static final long TIME = 5000;
+    SimpleFeatureSource fs;
+
+    ReferencedEnvelope bounds;
+
+    @Override
+    protected void setUp() throws Exception {
+        RendererBaseTest.setupVeraFonts();
+
+        bounds = new ReferencedEnvelope(0, 10, 0, 10, null);
+
+        // System.setProperty("org.geotools.test.interactive", "true");
+
+        SimpleFeatureTypeBuilder builder = new SimpleFeatureTypeBuilder();
+        builder.add("geom", Polygon.class);
+        builder.add("label", String.class);
+        builder.setName("labelFontShrinkSizeMin");
+        SimpleFeatureType type = builder.buildFeatureType();
+
+        GeometryFactory gf = new GeometryFactory();
+        SimpleFeature f1 =
+                SimpleFeatureBuilder.build(
+                        type,
+                        new Object[] {
+                            gf.createPolygon(
+                                    new Coordinate[] {
+                                        new Coordinate(4, 7),
+                                        new Coordinate(4, 9),
+                                        new Coordinate(7, 9),
+                                        new Coordinate(7, 7),
+                                        new Coordinate(4, 7)
+                                    }),
+                            "labelPoly A"
+                        },
+                        null);
+        SimpleFeature f2 =
+                SimpleFeatureBuilder.build(
+                        type,
+                        new Object[] {
+                            gf.createPolygon(
+                                    new Coordinate[] {
+                                        new Coordinate(2.5, 7),
+                                        new Coordinate(2.5, 8),
+                                        new Coordinate(4, 8),
+                                        new Coordinate(4, 7),
+                                        new Coordinate(2.5, 7)
+                                    }),
+                            "labelPoly B"
+                        },
+                        null);
+
+        MemoryDataStore data = new MemoryDataStore();
+        data.addFeature(f2);
+        data.addFeature(f1);
+        fs = data.getFeatureSource("labelFontShrinkSizeMin");
+    }
+
+    public void testFonstShrinkSize() throws Exception {
+        Style style = RendererBaseTest.loadStyle(this, "fontShrinkSize/fontShrinkSizeMin.sld");
```

Comment:
This file name starts with a lower case, the actual file starts with an uppercase, causing a build failure on Linux:

```
[INFO] Running org.geotools.renderer.lite.LabelFontShrinkSizeMinTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.005 s <<< FAILURE! - in org.geotools.renderer.lite.LabelFontShrinkSizeMinTest
[ERROR] testFonstShrinkSize(org.geotools.renderer.lite.LabelFontShrinkSizeMinTest)  Time elapsed: 0.001 s  <<< ERROR!
java.lang.NullPointerException
	at org.geotools.renderer.lite.LabelFontShrinkSizeMinTest.testFonstShrinkSize(LabelFontShrinkSizeMinTest.java:108)
```

Reply:
sld filename changed to be align with the one specified within code